lib/effects/rotate_effect.dart

```diff
@@ -26,7 +26,7 @@ class RotateEffect extends PositionComponentEffect {
     _originalAngle = component.angle;
     _peakAngle = _originalAngle + radians;
     _direction = _peakAngle.sign;
-    travelTime = (_peakAngle / speed).abs();
+    travelTime = (radians / speed).abs();
```

Comment:
this was a bug before right? nice catch!

Reply:
Hehe yeah, I was very confused why the times didn't add up for the sequence.

Reply:
Probably could add an entry on the changelog about this fix, what do you think?